src/MangaStream.ts

```diff
@@ -440,7 +445,7 @@ export abstract class MangaStream implements ChapterProviding, HomePageSectionsP
 
             // eslint-disable-next-line no-async-promise-executor
             promises.push(new Promise(async () => {
-                section.section.items = await this.parser.parseHomeSection($, section, this)
+                section.section.items = await this.parser.parseHomeSection($, section, this, this.baseUrl)
```

Comment:
What's the purpose for this and why should it be added? Has not been needed before?

Reply:
Some mangastream sites use src attributes like "https://google.com/images/abc.png" and others '/images/abc.png".
I thought it might be a nice feature to have an already implemented system might we add some of these mangastream sources in the future.
E.g. kissmanga